designsafe/static/scripts/data-depot/components/data-depot-listing/files-listing.template.html

```diff
@@ -143,11 +143,19 @@
                             Learn how to move files to a project
                         </a>
                     </p>
-                    <p class="lead" style="margin-top:20px" ng-if="$ctrl.editTags">
+                    <p class="lead" style="margin-top:20px" ng-if="$ctrl.editTags && !$ctrl.getType(listing.params.system) == 'Other'">
                         <i class="fa fa-file-text-o">&nbsp;</i><br/>
                         <a href="https://www.youtube.com/watch?v=lfBe04ihQUs&list=PL2GxvrdFrBlkwHBgQ47pZO-77ZLrJKYHV" target="_blank">
                             <i class="fa fa-question-circle" style="font-size: 18px;">&nbsp;</i>
                             Learn how to assign files to categories<br/>
+                        </a>  
+                    </p>
+                    <p class="lead" style="margin-top:20px" ng-if="$ctrl.editTags && $ctrl.getType(listing.params.system) == 'Other'">
```

Comment:
`listing.params.system` will not have the project type you're looking for regarding these projects. I don't actually think the project model is anywhere in the listing at the moment. We'll have to dig into the data bindings being passed into the FileLisingComponent. You'll see some one way data bindings that look like `showTags` or `editTags` This might be one way we could set a parameter like `missingFileMessage` that could be a bit more dynamic. Might discuss this in the next meeting.